src/dash-table/components/Filter/FilterOptions.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {FilterCase} from 'dash-table/components/Table/props';
+
+interface IFilterCaseButtonProps {
+    filterOptions: FilterCase;
+    toggleFilterOptions: () => void;
+}
+
+export default ({
+    filterOptions,
+    toggleFilterOptions
+}: IFilterCaseButtonProps) => (
+    <input
```

Comment:
Can we add a `title` attribute here as well? Something like 'Toggle filter case sensitivity'

Reply:
https://github.com/plotly/dash-table/pull/893/commits/d76ae04ef2badd066ef37437b8a6513f69927642